add more debouncer stuff to externs

danbeam opened the pull request:
/cc @jklein24 @rictic

danbeam committed:
add Polymer.dom.addDebouncer() to polymer-externs

jklein24 commented on polymer.externs.js:
```diff
@@ -722,6 +722,9 @@ Polymer.dom = function(nodeOrEvent) {};
 
 Polymer.dom.flush = function() {};
 
+/** @param {!function()} debouncer */
```
I don't think this param type is correct. See https://github.com/Polymer/polymer/blob/master/src/lib/debounce.html#L33.

danbeam committed:
addDebouncer -> _debounceTemplate

https://github.com/Polymer/polymer/blob/2c6161c253296f9dd098e6c82af24fb328f0ed5f/src/lib/template/templatizer.html#L180

danbeam committed:
remove addDebouncer

danbeam committed:
damn \n

danbeam committed:
add back Polymer.Debouncer (silly closure compiler is not deterministic)

jklein24 commented on polymer.externs.js:
```diff
@@ -722,6 +722,24 @@ Polymer.dom = function(nodeOrEvent) {};
 Polymer.dom.flush = function() {};
 
+/** @constructor */
+Polymer.Debouncer;
```
... = function() {};

danbeam committed:
jklein24 nit

jklein24 commented:
LGTM :+1:
